GSG-G7/mental-snapp
# Heat Map

issue: #17 Heat Map

## What did I do?
- **Installed some packages**
  - removed the unneeded apexcharts packages 
  - installed the cal-heatmap package 
    - for the heat-map
  - the react tooltip to display the tooltips 
  - used moment to display the date and time in a good way

- **Used the cal heatmap to create the heat map**
  - if you click on an item in the heatmap it should display the date of that day and how many journals are in it 
- **Used fake data for:**
   - the heatmap: a data that contains  each day in the current month and how many journals in it (should come from our DB)
   - the journals data: let's say the user has clicked on a date with journals in it then we need to show them so I put some journals to try how the page will look and improved the style of it 

- **Made some functions**
   - `handleClick`: which should handle each click on any of the squares in the heatmap(should be implemented later)
   - `toolTip`: it handles what to display in the tooltip

- **Styled the page**
   - in the css file you'll see all the styles 
   - I made the page responsive so it looks good on all screen sizes 

## How to test?
- first `npm i` to install the new packages 
- Just go to the `/heat-map` route and you shall see the page with journals on 
- You can remove the journals from the state(put an empty array) so you can see it without journals
  ```js
  journals = []
  ```
